doc/source/tune/examples/tune-pytorch-lightning.ipynb

```diff
@@ -581,6 +581,7 @@
     "## More PyTorch Lightning Examples\n",
     "\n",
     "- {ref}`Use LightningTrainer for Image Classification <lightning_mnist_example>`.\n",
+    "- {ref}`Use LightningTrainer with Ray Data and Batch Predictor <lightning_advanced_example>`\n",
```

Comment:
Can we also link it at the end of the basic example in the `What’s next?` section?

Reply:
sg. Thanks matt! I'll address these comments in a separate PR.